understanding/20/error-identification.html

```diff
@@ -72,21 +72,18 @@ <h2>Intent of Error Identification</h2>
          that user agents or assistive technologies can use to identify an error and provide
          error information to the user. For example, certain technologies can specify that
          the user's input must not fall outside a specific range, or that a form field is required.
-         Currently, few technologies support this kind of programmatic information, but the
-         Success Criterion does not require, nor prevent it.
-         
+         This type of programmatic information is not required for this success criterion, but may be covered
```

Comment:
While we are at it, would it be worth covering the frequent case of inline error handling, e.g. mentioning the recommended position of error messages (if placed above the field these would easiliy be missed in SR reading mode)? Information could also indicate that these messages should be programmatically available (e.g. by inserting them in a programmatically linked `label` or linking them with `aria-describedby`). If I get the gist of the reference to 4.1.2 you think that failure to programmatically link error messages should be reported elsewhere, like the existence of the required attribute when required is visually indicated?

Reply:
> While we are at it, would it be worth covering the frequent case of inline error handling, e.g. mentioning the recommended position of error messages (if placed above the field these would easiliy be missed in SR reading mode)?

If you want to open a new issue to include that, it's welcome, but I think it's important that we maintain a process of trying not to increase scope in PRs that have been put in front of the working group.